app.json

```diff
@@ -18,41 +18,9 @@
       "description": "Mongo Password (see MONGO_USERNAME)",
       "value": "Squidex123"
     },
-    "EVENTSTORE__TYPE": {
-      "description": "EventStore Type",
-      "value": "MongoDB"
-    },
     "IDENTITY__ADMINEMAIL": {
```

Comment:
You can also remove the admin email because the initial admin can be configured with a setup screen.